### DISCUSSION TITLE
Is there an event for Security Alerts?

### DISCUSSION THREAD

**Discussion by arnested** [community/community#26155] [Actions]
<p>Hi,</p>
<p>Is there an event that gets triggered when a Security Alert is created?</p>
<p>Kind regards,</p>
<p>Arne</p>

**Comment by eddiechenms** [comment]
<p>Yes, there is a <a href="https://developer.github.com/v3/activity/events/types/#repositoryvulnerabilityalertevent" rel="nofollow noopener">RepositoryVulnerabilityAlertEvent</a>which is triggered when a sceurity alert is created, dismissed or resolved. But it cannot be used to trigger a workflow as it hasn’t been added to <a href="https://help.github.com/en/github/automating-your-workflow-with-github-actions/events-that-trigger-workflows" rel="nofollow noopener">Events that trigger workflows</a>.</p>

**Comment by arnested** [comment]
<p>Thank you, <a class="mention" href="/eddiechenms">@eddiechenms</a>.</p>
<p>So an event exists. It just cant be used with GitHub Actions?</p>
<p>Triggering a workflow was exactly I was/am hoping for.</p>
<p>Someone please add it to a wishlist :slight_smile:</p>

**Comment by chrispat** [comment, marked as answer]
<p>Right now we do not support that event as viewing security alerts is more restricted than simjply view or editing files.  If we enabled that event anyone who could get a file into the repos could start essentially viewing security alerts.</p>

**Comment by arnested** [comment]
<p>Thank you for the explanation, <a class="mention" href="/chrispat">@chrispat</a>.</p>
<p>I can see how that could be problematic.</p>
<p>We’ll find another way to do what we wanted to do (which is create a Jira ticket for new security alerts).</p>
<p>Kind regards,</p>
<p>Arne</p>

**Comment by aThorp96** [comment]
<p>Is it currently possible to subscribe to these events?</p>
